src/redisraft.c

```diff
@@ -1334,6 +1527,93 @@ static int cmdRaftRandom(RedisModuleCtx *ctx,
     return REDISMODULE_OK;
 }
 
+static int cmdRaftScan(RedisModuleCtx *ctx,
+                         RedisModuleString **argv, int argc)
+{
+    RedisRaftCtx *rr = &redis_raft;
+
+    if (argc < 3) {
+        RedisModule_WrongArity(ctx);
+        return REDISMODULE_OK;
+    }
+
+    char cursor[15] = {0};
```

Comment:
Why this is `15`? Can we expect more? 

Btw, after calling `strncpy()` for this buffer, please add `\0` at the end. Currently, as you initialize it with zeros, we don't run into problems but it is risky this way if we remove that initialization line one day. It crashed for `slot_str` here previously (Josh & I merged a fix for it).

Reply:
I believe 15 was wider than the max length of an unsigned long long

Reply:
`unsigned long long` is 8 bytes, it's wider than that. If I don't know the length, I use buffer size 32 or even 256.